lib/system.nim

```diff
@@ -65,17 +65,21 @@ const
 {.push hints: off.}
 
 type
-  Ordinal* {.magic: Ordinal.}[T]
+  Ordinal* {.magic: Ordinal.}[T] ## Generic ordinal type. Includes integer,
+                                 ## bool, character, and enumeration types
+                                 ## as well as their subtypes. Note `uint`
+                                 ## and `uint64` are not ordinal types for
+                                 ## implementation reasons
   `ptr`* {.magic: Pointer.}[T] ## built-in generic untraced pointer type
   `ref`* {.magic: Pointer.}[T] ## built-in generic traced pointer type
 
-  `nil` {.magic: "Nil".}
+  `nil` {.magic: "Nil".}  ## special type for pointers, references, and other `nil`-able types
```

Comment:
No, that's just an implementation artifact. IIRC I had to add this for symbol files.